model/import/Validator/LineValidator.php

```diff
@@ -48,11 +49,23 @@ public function validate(array $content, TemplateInterface $csvTemplate): void
                 $validator = $this->getValidatorMapper()->getValidator($name);
                 if ($validator) {
                     try {
-                        $validator->validate($content[$headerRegex] ?? null, $rules, $content);
+                        $validatedValue = $content[$headerRegex] ?? null;
+
+                        $loggedValue = is_null($validatedValue) ?
+                            'NULL' :
+                            sprintf('"%s"', $validatedValue);
+
+                        $validator->validate($validatedValue, $rules, $content);
+
+                        $logger->debug(sprintf('Tabular import: successful validation on %s by "%s" validator', $loggedValue, $name));
                     } catch (RecoverableLineValidationException $exception) {
                         $warnings->addWarning(0, sprintf($exception->getMessage(), $headerRegex), $headerRegex);
+
+                        $logger->warning(sprintf('Tabular import: failed validation on %s by "%s" validator', $loggedValue, $name));
                     } catch (InvalidImportException | InvalidCsvImportException $exception) {
                         $warnings->addError(0, sprintf($exception->getMessage(), $headerRegex), $headerRegex);
+
+                        $logger->error(sprintf('Tabular import: failed validation on %s by "%s" validator', $loggedValue, $name));
```

Comment:
All the logs here should be debug, cause this is a controlled error. We should only log warning and error in case there is something unexpected or out of normal.

Wdyt?

Ping @boajer @voknil